### ISSUE TITLE
SAS proof of concept, metadata questions

### ISSUE THREAD

**Issue by couthcommander** [vubiostat/redcapAPI#285]
I've written a function that uses `exportBulkRecords` to create CSV files and accompanying script to import to SAS. I'm still not sure this is a thing that's needed, but it could use a review. As this relies heavily on `assembleCodebook`, I'll note a few things I didn't expect:
* the codebook/metadata does not find REDCAP_SYSTEM_FIELDS or *_complete fields
* related to system fields, I can build categorical variables with labels using `.castRecords_getSystemCoding`, `.assembleCodebook_unexpanded`, and `fieldChoiceMapping`
* should `assembleCodebook` include "field_label"?

https://gist.github.com/couthcommander/152e85f0fa9779034ea7c22cf6adc8ce

**Comment by nutterb** [comment]
These are all oversights from the original design. In fact, it's clear the *_complete fields were intended in the original design. I just...didn't do it?

Anyhow, to do list:

- [ ] Add *_complete fields
- [ ] Add system fields
- [ ] Add field labels

**Comment by couthcommander** [comment]
I've updated my gist to reflect PR #296 and it's working well (much simpler). It does still need the "redcap_repeat_instance" system field.

**Comment by couthcommander** [comment]
Yes, I'll close. SAS/SDTM exports can be a standalone feature request if desired. The referenced gist is a good starting point on either.

**Comment by spgarbet** [comment]
After discussion, we'll include this an experimental feature.
